oximeter/collector/src/self_stats.rs

```diff
@@ -9,51 +9,21 @@
 use crate::ProducerEndpoint;
 use oximeter::types::Cumulative;
 use oximeter::types::ProducerResultsItem;
-use oximeter::Metric;
 use oximeter::MetricsError;
 use oximeter::Sample;
-use oximeter::Target;
 use reqwest::StatusCode;
 use std::borrow::Cow;
 use std::collections::BTreeMap;
-use std::net::IpAddr;
 use std::time::Duration;
-use uuid::Uuid;
+
+oximeter::use_timeseries!("oximeter-collector.toml");
```

Comment:
Should this file exist in the PR? Or is it getting auto-generated in some way I'm not seeing?

Reply:
It should, if I remembered to add it :) Thanks, adding now.

Reply:
Added in f101ca03b